### DISCUSSION TITLE
如何关闭client的连接 再一个进程里 多次 连接不同的config service

### DISCUSSION THREAD

**Discussion by dakai6677** [apolloconfig/apollo#3742] [Q&A]
标题内容 感谢回复

**Comment by nobodyiam** [comment]
目前的 sdk 实现不支持这个场景，是什么场景需要这种能力？

**Reply by dakai6677** [reply]
主要想校验 不同的得Apollo环境 是不是安装 运行正常 昨天找到admin open api 有类似的接口  想问一下 调用open api 的接口 可不可以使用一个 通用的token 在多个 实例上进行接口调用

**Reply by dakai6677** [reply]
就是 只需要传 portal的IP跟端口 使用一个 通用的 token 类似管理员默认的token 调用多个Apollo实例环境

**Reply by Anilople** [reply]
一个token可以管理多个环境

![image](https://user-images.githubusercontent.com/15523186/120631007-f71f5c00-c499-11eb-9f93-2f1b5e1588e0.png)

授权的时候，把授权类型设置为App

或者环境勾选多个就可以了

**Reply by dakai6677** [reply]
好的 谢谢 已经解决了 麻烦了
